errors/sharp-missing-in-production.md

```diff
@@ -2,13 +2,34 @@
 
 #### Why This Error Occurred
 
-The `next/image` component's default loader uses [`squoosh`](https://www.npmjs.com/package/@squoosh/lib) because it is quick to install and suitable for a development environment. For a production environment using `next start`, it is strongly recommended you install [`sharp`](https://www.npmjs.com/package/sharp) by running `yarn add sharp` in your project directory.
+The `next/image` component's default loader uses [`squoosh`](https://www.npmjs.com/package/@squoosh/lib) because it is quick to install and suitable for a development environment.
+
+- For a production environment using `next start`, it is strongly recommended you install [`sharp`](https://www.npmjs.com/package/sharp).
 
 You are seeing this error because Image Optimization in production mode (`next start`) was detected.
 
+- For a production environment using `output: "standalone"`, you must install [`sharp`](https://www.npmjs.com/package/sharp).
+
+You are seeing this error because Image Optimization in standalone mode (`output: "standalone"`) was detected.
+
 #### Possible Ways to Fix It
 
-- Install `sharp` by running `yarn add sharp` in your project directory and then reboot the server by running `next start` again
+- Install `sharp` by running one of the following commands in your project directory:
+
+```bash
+npm i sharp
+```
+
+```bash
+yarn add sharp
+```
+
+```bash
+pnpm add sharp
+```
+
+Then, reboot the server by running `next start` again, or if you have `output: "standalone"` enabled, rebuild your project with `next build` and reboot the server with `node server.js`.
+
 - If `sharp` is already installed but can't be resolved, set the `NEXT_SHARP_PATH` environment variable such as `NEXT_SHARP_PATH=/tmp/node_modules/sharp next start`
```

Comment:
This line is using `next start` but should probably show the variant for `node server.js`. Or maybe we should just use `export NEXT_SHARP_PATH=""` as a separate line so its set for both build and start.